src/arrays/what-is-practicum.js

```diff
@@ -1,60 +1,80 @@
+import image from '../images/practicum-logo.png';
+
 const practicum = [
```

Comment:
I just have to say I like storing page content in arrays like this. What a wonderful way to remove all the fluff from the component files. So so nice.

Reply:
it definitely makes editing content easier. like fixing a typo